datalad/distributed/create_sibling_gitlab.py

```diff
@@ -395,6 +407,9 @@ def _proc_dataset(refds, ds, site, project, remotename, layout, existing,
             "Unknown site access '{}' given or configured, "
             "known ones are: {}".format(access, known_access_labels))
 
+    pathsep = ds.config.get("datalad.gitlab-default-pathseparator", None) or "-"
+    project_stub = \
+        ds.config.get("datalad.gitlab-default-projectname", None) or "project"
```

Comment:
```suggestion
    project_stub = \
        ds.config.get("datalad.gitlab-default-projectname", "project")
```

Reply:
```suggestion
    project_stub_cfg = "datalad.gitlab-default-projectname"
    project_stub = ds.config.get(
        project_stub_cfg,
        refds.config.get(project_stub_cfg, "project"))
```

Same reasoning as above